neuro_extras/main.py

```diff
@@ -1000,10 +1000,18 @@ async def _run_copy_container(src_cluster: str, src_path: str, dst_path: str) ->
     ]
     cmd = " ".join(args)
     print(f"Executing '{cmd}'")
-    subprocess = await asyncio.create_subprocess_shell(cmd)
+    subprocess = await asyncio.create_subprocess_shell(
+        cmd, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE
+    )
+    stdout, stderr = await subprocess.communicate()
```

Comment:
I'm against this change: the user should see live what's going on, but not after (if) the command failed.